python/paddle/fluid/layers/learning_rate_scheduler.py

```diff
@@ -360,13 +362,13 @@ def cosine_decay(learning_rate, step_each_epoch, epochs):
         Variable: The decayed learning rate.
 
      Examples:
```

Comment:
can't find this part in preview pic

Reply:
> can't find this part in preview pic

Do we have a better self-check process? for now, the developer will repeat "compile -> preview -> rewrite ->compile", most caused by careless though,  I suggest supporting a tool to preview badly, like just copy these comments to the pasteboard, then preview the comments online without compiling the whole program

Reply:
> > can't find this part in preview pic
> 
> Do we have a better self-check process? for now, the developer will repeat "compile -> preview -> rewrite ->compile", most caused by careless though, I suggest supporting a tool to preview badly, like just copy these comments to the pasteboard, then preview the comments online without compiling the whole program

we will consider it, thanks for your suggestion